app/views/pages/_home_page_bottom_section.html.erb

```diff
@@ -41,10 +75,15 @@
         <p>
           Private and Secure
         </p>
-        <p class="text--help">
+        <p class="text--help with-padding-large">
           Your information is kept private and secure. MichiganBenefits
           maintains strict security guidelines to protect your privacy and indentity.
           For details, read the full <%= link_to "Privacy Policy", privacy_path, target: "_blank" %>.
+        </p>
+        <%= link_to step_path(StepNavigation.first), class: "button button--cta button--fully-width" do %>
+          Apply now
```

Comment:
we want this link to go to a different path depending on if it is an office page.

One way to do this would be to pass a var to the `home_page_buttom` partial in each view 

eg, on `index`, `clio` and `union` it would be these 3 (one line in each file):

```
<% render "home_page_bottom_section", office_location: "" %>
<% render "home_page_bottom_section", office_location: "clio" %>
<% render "home_page_bottom_section", office_location: "union" %>
```

and then here it would be

```
<%= link_to step_path(StepNavigation.first, office_location: office_location) class: "button button--cta button--fully-width" do %>
```

Reply:
@jessieay That's a great point. I'll add this in!